language_detector/main.py

```diff
@@ -2,10 +2,24 @@
 
 """This is the entry point of the program."""
 
-from languages import LANGUAGES
+from .languages import LANGUAGES
+
+def detectInOneLanguage(text,lang):
```

Comment:
Did you consider the case of cognate words in the text? For example, German and English have some cognates words (all, has, die, so). Case you put a german text and it get the 'all' word, you could get 'English' as return of the function.